docs/tutorial/tutorial-0.rst

```diff
@@ -75,6 +76,8 @@ Next, install Toga into your virtual environment:
 
       (venv) C:\...>pip install --pre toga
 
+If you get other errors, please check that you followed [prerequisites](https://github.com/beeware/toga#prerequisites) instructions.
```

Comment:
This doesn't look like reStructured Text for the external link, did you accidentally use Markdown instead?

Reply:
indeed.